core-web/libs/edit-content/src/lib/feature/edit-content/edit-content.layout.component.spec.ts

```diff
@@ -17,7 +20,8 @@ const createEditContentLayoutComponent = (params: { contentType?: string; id?: s
             {
                 provide: ActivatedRoute,
                 useValue: { snapshot: { params } }
-            }
+            },
+            { provide: DotFormatDateService, useClass: DotFormatDateServiceMock }
```

Comment:
Why are we adding this here if we are not adding it to the component?

Reply:
Layout render the aside and aside needs this service to work
We can mock the provider without the useClass, using mockProvider()
But i think we must to mock it to make the test work